### ISSUE TITLE
Min length condition for server side queries

### ISSUE THREAD

**Issue by CarbonFactory** [MoonStorm/trNgGrid#164]
Can you add an option wherein we can specify the number of characters that should be entered in the search filter input, only after which the "on-data-required" callback function will be invoked ?

i.e if the value for the above parameter is 4, then only if 4 or more characters are entered, the function will be invoked. I can handle this inside the function itself but will be a messy one.

**Comment by MoonStorm** [comment]
I postponed an answer to this feature request since I'm aware it's gonna open a can of worms. People might start asking for different types of validations.

What I'd suggest you do is to attempt to achieve what you need by overriding the templates for the column filter and global filter. Check the source code for the original ones.

**Comment by CarbonFactory** [comment]
I get what you mean :grin:  Will try the workaround
